### DISCUSSION TITLE
Command alias that's not listed

### DISCUSSION THREAD

**Discussion by ZakShearman** [Minestom/Minestom#1139] [Help] [answered]
I would like to add an alias for my /reply command (/r) but not have it shown to players. 

IMO a lot of these can just bloat the command list - making it harder for players to get started - so I'd prefer they not be shown. Is this possible?

**Comment by PlugDev-class** [comment, marked as answer]
Theoretically you could add an PlayerCommandEvent.
There you don't need to add an alias, like you'd need for commands.
- The only Problem hereby is, that your command will be displayed as invalid.

Good luck

**Reply by ZakShearman** [reply]
Yh I've now realised that having the command being displayed as invalid is a much worse experience. Thx <3
